src/main/migrations/20180508093508_init.js

```diff
@@ -94,6 +94,7 @@ exports.up = function(knex, Promise) {
 			table.integer('isCustom').notNullable().defaultTo(0)
 			table.integer('createdAt').notNullable()
 			table.integer('updatedAt')
+			table.string('type').notNullable()
```

Comment:
removed in other branch, also removed from seed data

Reply:
@altninja I added it back because it was failing without it. I will try to remove it and fix where it is requiring it. cheers